uaclient/tests/test_system.py

```diff
@@ -679,6 +690,43 @@ def test_get_platform_info_error_no_version(self, m_parse):
                     "version": "22.10 LTS (Kinetic Kudu)",
                 },
             ),
+            (
+                {
+                    "NAME": "Ubuntu",
+                    "VERSION": "22.04 LTS",
+                    "VERSION_CODENAME": "Jammy",
+                },
+                "amd64",
+                "kernel-ver4",
+                {
+                    "arch": "amd64",
```

Comment:
Since the real errors came from derivatives like mint/popos and such - should we add test data based on those as well here?

Reply:
We can, but our goal is not to prevent those releases from failing here, but failing gracefully if we can't extract that information. The refactor in this part of the code is just so we can better identify in the logs the cause of failures. By that, I mean that we don't want to support those releases with this PR, just allow the code to fail gracefully. This is achieved by the new `try..except` block on the `esm_cache` module that is now running on those releases. But let me know if you disagree

Reply:
I agree with the solution, I just wanted to see a full content block as it would appear in one of the derivatives in the test data.